static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx

```diff
@@ -36,10 +36,20 @@ export function SpanEvidenceKeyValueList({
         issueType === IssueType.PERFORMANCE_N_PLUS_ONE_DB_QUERIES
           ? t('Repeating Span')
           : t('Offending Span'),
-      value: getSpanEvidenceValue(offendingSpan),
+      value: getSpanEvidenceValue(offendingSpans[0]),
     },
   ];
 
+  const problemParameters = getProblemParameters(offendingSpans);
+
+  if (problemParameters && issueType === IssueType.PERFORMANCE_N_PLUS_ONE_API_CALLS) {
```

Comment:
FYI (you may already be aware) this logic will need to be on the backend soon once the issue platform team pushes out occurrences: https://github.com/getsentry/sentry/pull/42366

Reply:
Whoa, I had no idea this was happening, thanks for the heads-up! Please let the Performance team know when this work gets merged, we have a bunch of new issue types in the hopper 🙏🏻